src/jdk.compiler/share/classes/com/sun/tools/javac/processing/JavacProcessingEnvironment.java

```diff
@@ -1642,7 +1642,7 @@ public void visitClassDef(JCClassDecl node) {
                     List<? extends RecordComponent> recordComponents = node.sym.getRecordComponents();
                     for (RecordComponent rc : recordComponents) {
                         List<JCAnnotation> originalAnnos = rc.getOriginalAnnos();
-                        originalAnnos.stream().forEach(a -> visitAnnotation(a));
+                        originalAnnos.forEach(this::visitAnnotation);
```

Comment:
while not wrong, this is not covered in the description of the PR

Reply:
Reverted